### PR TITLE
stars was created

### PR DESCRIPTION
**Pull request by blcts**
- [DEMO LINK](https://blcts.github.io/layout_stars/)
- [TEST REPORT LINK](https://blcts.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.body {
+  margin: 0;
+
+}
```
Comment (VETALEK) [inline src/style.css:4]:
```suggestion
.body {
  margin: 0;
}
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 20px;
+  height: 16px;
```
Comment (VETALEK) [inline src/style.css:14]: the stars are too wide, you may use `margin` or `gap` to make space between them, but not a `width` 😉

image

**Review by VETALEK — CHANGES_REQUESTED** [review]
Good start 🔥

**Review by VETALEK — CHANGES_REQUESTED** [review]
almost 💪

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```
[the remark this answers is not in the document]
Comment (solaryasha) [reply src/style.css:21]: Зараз взагалі немає відступів між зірками

**Review by solaryasha — CHANGES_REQUESTED** [review]

**Review by VETALEK — APPROVED** [review]
🤩

**Pull request closed without merging**